PC/layout/main.py

```diff
@@ -66,18 +66,6 @@
 TOOLS_DIRS = FileNameSet("scripts", "i18n", "pynche", "demo", "parser")
 TOOLS_FILES = FileSuffixSet(".py", ".pyw", ".txt")
 
-def copy_if_modified(src, dest):
```

Comment:
Did you mean to revert your changes to this file?

Reply:
No.  I meant to rebase on top of the layout changes in the other PR. I'll fix this

Reply:
fixed